test/sharness/t0139-swarm-rcmgr.sh

```diff
@@ -81,11 +81,13 @@ test_expect_success 'connected: swarm stats all working properly' '
 test_expect_success 'ResourceMgr enabled: swarm stats' '
   ipfs swarm stats all --enc=json | tee json &&
   jq -e .System.Memory < json &&
-  jq -e .System.NumConnsInbound < json &&
-  jq -e .System.NumConnsOutbound < json &&
-  jq -e .System.NumFD < json &&
-  jq -e .System.NumStreamsInbound < json &&
-  jq -e .System.NumStreamsOutbound < json &&
+  jq -e .System.Conns < json &&
+  jq -e .System.ConnsInbound < json &&
+  jq -e .System.ConnsOutbound < json &&
+  jq -e .System.FD < json &&
+  jq -e .System.StreamsInbound < json &&
+  jq -e .System.StreamsOutbound < json &&
+  jq -e .System.Streams < json &&
```

Comment:
```suggestion
  jq -e .System.FD < json &&
  jq -e .System.Conns < json &&
  jq -e .System.ConnsInbound < json &&
  jq -e .System.ConnsOutbound < json &&
  jq -e .System.Streams < json &&
  jq -e .System.StreamsInbound < json &&
  jq -e .System.StreamsOutbound < json &&
```

nit: did more "logical ordering" that we have in the code as well.